visualization/Readme.md

```diff
@@ -32,7 +32,7 @@ This page enables the analysis of model answers based on different parameters. I
 - **Run whole dataset** mode lets you launch the generation with chosen parameters on the entire dataset. Results are saved in `visualization/results/output-greedy.jsonl` and `visualization/results/metrics-greedy.jsonl`. If the "use random seed range" flag is enabled, each answer will be sampled with multiple random seeds in the range from `start_random_seed` to `end_random_seed`. After generation is done, you can review the results on the "Analyze" page. The parameters used for dataset launch are also recorded in the `visualization/results/parameters.jsonl` file and displayed on the "Analyze" page.
 
 ## Analyze page
-To use the Analyze page, specify paths to the generations you want to use (if not obtained through the "Inference" page). You can pass parameters via the command line with `++visualization_params.model_prediction.generation1='/some_path/generation1/output-greedy.jsonl'` or add them in an additional config file.
+To use the Analyze page, specify paths to the datasets you want to use (if not obtained through the "Inference" page). You can pass parameters via the command line with `++visualization_params.model_prediction.dataset1='/some_path/generation1/output-greedy.jsonl'` or add them in an additional config file.
```

Comment:
I'm a bit confused by usage of "dataset" vs "generation" in this doc. Can you just pick one and consistently use it everywhere?